### PR TITLE
Add temporary limits to 2 wt

### PR DESCRIPTION
**Pull request by Meklo**

### CODE REVIEW

File: src/components/refactor/dialogs/two-windings-transformer-creation/two-windings-transformer-creation-dialog.js [hunk 1]
```diff
@@ -575,6 +619,9 @@ const TwoWindingsTransformerCreationDialog = ({
                     ...twt[PHASE_TAP_CHANGER],
                 };
             }
+
+            console.log(currentLimits1);
+            console.log(currentLimits2);
```
Comment (YenguiSeddik) [inline src/components/refactor/dialogs/two-windings-transformer-creation/two-windings-transformer-creation-dialog.js:624]: I think those should be removed

File: src/components/refactor/dialogs/two-windings-transformer-creation/two-windings-transformer-pane/two-windings-transformer-pane.js [hunk 2]
```diff
@@ -208,14 +190,14 @@ const TwoWindingsTransformerPane = ({
                     </h3>
                 </Grid>
             </Grid>
-            <Grid container spacing={2}>
+            <Grid container spacing={2} xs={8}>
```
Comment (YenguiSeddik) [inline src/components/refactor/dialogs/two-windings-transformer-creation/two-windings-transformer-pane/two-windings-transformer-pane.js:193]: I think when you use xs, the grid should be <Grid container **item** spacing={2} xs={8}> or <Grid **item** spacing={2} xs={8}> or there will be warnings in the console

File: src/components/refactor/dialogs/two-windings-transformer-creation/two-windings-transformer-creation-dialog.js (hunk 1 above)
Comment (Meklo) [reply src/components/refactor/dialogs/two-windings-transformer-creation/two-windings-transformer-creation-dialog.js]: Done, there's always one or two debug message that slip through the net..

File: src/components/refactor/dialogs/two-windings-transformer-creation/two-windings-transformer-pane/two-windings-transformer-pane.js (hunk 2 above)
Comment (Meklo) [reply src/components/refactor/dialogs/two-windings-transformer-creation/two-windings-transformer-pane/two-windings-transformer-pane.js]: Done, I went for the first option and applied it to all the places it's relevant

File: src/components/refactor/dialogs/two-windings-transformer-creation/limits-pane/two-windings-transfomer-limits-pane-utils.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
+/**
```
Comment (dbraquart) [inline src/components/refactor/dialogs/two-windings-transformer-creation/limits-pane/two-windings-transfomer-limits-pane-utils.js:1]: These 2 new files are strictly the same than in **line-creation/line-limits-pane** folder. I didn't expect that.
I think we have to use 2 common files for both line and 2wt.
Maybe in a new folder **dialog/limits-table** ?

your opinion ?

**Review by dbraquart — CHANGES_REQUESTED** [review]
tests: ok
console: ok
review: a possible code factorisation

File: src/components/refactor/dialogs/two-windings-transformer-creation/limits-pane/two-windings-transfomer-limits-pane-utils.js (hunk 3 above)
Comment (Meklo) [reply src/components/refactor/dialogs/two-windings-transformer-creation/limits-pane/two-windings-transfomer-limits-pane-utils.js]: After pondering it there's no real drawbacks to it, I've implemented it and taken care of applying the last changes you made to line limits pane

Comment (dbraquart) [reply src/components/refactor/dialogs/two-windings-transformer-creation/limits-pane/two-windings-transfomer-limits-pane-utils.js]: yes, but it should be the same

**Review by YenguiSeddik — APPROVED** [review]

**Pull request merged**